src/main/java/com/strangeone101/holoitemsapi/enchantment/EnchantmentListener.java

```diff
@@ -104,4 +105,32 @@ public void onProjectileLaunch(ProjectileLaunchEvent event) {
             }
         });
     }
+
+    /**
+     * Handles PlayerToggleSneak enchantments.
+     * Only does armor slots for now.
+     *
+     * @param event The ProjectileLaunchEvent
+     */
+    @EventHandler(ignoreCancelled = true)
+    public void onPlayerToggleSneak(PlayerToggleSneakEvent event) {
+        final var player = event.getPlayer();
+        final var inventory = player.getInventory();
+
+        for (int i = 0; i < inventory.getArmorContents().length; i++) {
+            ItemStack item =  player.getInventory().getArmorContents()[i];
+
+            if (item == null){
+                return;
```

Comment:
This looks like it's supposed to be a `continue` instead of a `return`. If it returns here, some armor is skipped if a previous slot is empty.

Reply:
Yup, that's my bad.